src/api/request.js

```diff
@@ -0,0 +1,43 @@
+const productsURL = 'https://mate-academy.github.io/react_phone-catalog/api';
```

Comment:
```suggestion
const PRODUCTS_URL = 'https://mate-academy.github.io/react_phone-catalog/api';
```
use upper snake case for static text constants